src/pages/workspaces/workspace/Data.js

```diff
@@ -625,23 +611,50 @@ class EntitiesContent extends Component {
         })
       }, [
         icon('copy-to-clipboard', { style: { marginRight: '0.5rem' } }),
-        'Copy to Clipboard'
+        'Copy Page to Clipboard'
       ]),
       copying && spinner()
     ])
   }
 
-  renderToolButton() {
+  renderSelectedRowsMenu(columnSettings) {
+    const { workspace, entityKey } = this.props
     const { selectedEntities } = this.state
-
-    return h(ButtonPrimary, {
-      style: { marginRight: '1rem' },
-      disabled: _.isEmpty(selectedEntities),
-      tooltip: 'Open the selected data',
-      onClick: () => this.setState({ showToolSelector: true })
-    }, [
-      'Open with...'
-    ])
+    const isSet = _.endsWith('_set', entityKey)
+    const disabled = _.isEmpty(selectedEntities)
```

Comment:
If all of the items in the menu are disabled, perhaps we should disable the menu itself?